### DISCUSSION TITLE
You did something! - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#36] [General]
# You did something! - Alan Pope's blog

This is the third in a series of Tales From Tech Support. Some stories from the past featuring broken computers and even more broken tech support operatives - mostly me.
In the early 1990s, I worked as a contractor for a large, well-established accounting firm. In It’s MY monitor, I told the story of dealing with an angry Partner. Today’s story is worse, as I’m dealing with the Personal Assistant of a Partner.

https://popey.com/blog/2023/09/you-did-something/

**Comment by dwvisser** [comment]
I have had this pain with actual family members, which is why I have quit the family tech support biz except for those with a track record of actually wanting my help. 🙂
